src/components/Content/TitledCategoriesTable.css

```diff
@@ -0,0 +1,5 @@
+
+.title {
+  margin: 30px 0 50px;
+  font-size: 40px;
+}
```

Comment:
I would prefer a `text-align: center;` here, because all other content is centered on the categories page.
Also I would decrease font-size to sth like `35px`, 40px can be a little bit loo large for longer titles and smaller screens.
From personal preference also `margin: 10px 0 30px;` But this can be discussed 😄 
But the styling of the headings should all match, so we should also adjust the "Where am I?"-Heading of the landing page imo.

Reply:
centered the text and adjusted the style to the "Where are you?" heading